src/app/components/preview-card/preview-card.component.scss

```diff
@@ -0,0 +1,60 @@
+.mat-card {
+  position: relative;
+  margin: 20px 10px;
+  transition: 0.2s;
+  transition-property: box-shadow;
+
+  &:hover, &:focus {
+    box-shadow: 0 5px 11px 0 rgba(0, 0, 0, 0.18),
+      0 4px 15px 0 rgba(0, 0, 0, 0.15);
+  }
+}
+
+.mat-card-title {
+  margin-left: -16px; // align header tile with content (mat-card-text not responsible)
+}
+
+.tags-box {
+  display: inline-table;
+}
+
+.tags {
+  background-color: #e55a17; // #8e92de
+  color: #fff;
+  border-radius: 20px;
+  margin: 8px 8px 8px 0;
+}
+
+.info {
+  display: grid;
+  grid-template-columns: 3fr 1fr;
+  grid-column-gap: 2em;
+  align-items: center;
+
+  .intros {
+    justify-self: left;
+  }
+
+  .dates {
+    justify-self: right;
+    text-align: right;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+
+    b {
+      text-align: right;
+      padding-right: 10px;
+    }
+  }
+}
+
+.synapse-icon {
+  position: absolute;
+  background-image: url("https://www.synapse.org/images/logo.svg");
+  background-size: cover;
+  right: 26px; // 16 + 10 align card
+  top: 26px;
+  transform: scale(1.5);
+  z-index: 2;
+}
```

Comment:
We should probably add some `@media` rules for mobile viewing and/or when the screens are smaller. Otherwise, something like this could happen:

**Before horizontal scroll**
<img width="539" alt="Screen Shot 2021-06-30 at 7 29 12 PM" src="https://user-images.githubusercontent.com/9377970/124056306-48a80180-d9da-11eb-9a81-7467cbff7f6e.png">

**After**
<img width="544" alt="Screen Shot 2021-06-30 at 7 29 06 PM" src="https://user-images.githubusercontent.com/9377970/124056157-0da5ce00-d9da-11eb-8f90-b38972363f98.png">

Reply:
Yeah, sure. I haven't made it responsive for now. The filter and news sections are temporarily set to have general view of challenge box location, but we should design better based on the breakpoints (maybe already set in the current codebase) to make responsive to smaller screen later.

Reply:
@rrchai to open a new ticket describing this issue before including the link here and resolving the conversation.